### PR TITLE
MM-41036: Render update request post in threads

### PR DESCRIPTION
**Pull request by lieut-data**
#### Summary
The update request custom post failed to render in the threads view because it depended on `currentPlaybookRun`, of which there isn't one outside of a playbook run channel.

**Before:**
CleanShot 2022-01-26 at 16 27 15@2x

**After:**
CleanShot 2022-01-26 at 16 27 31@2x

#### Ticket Link
Fixes: https://mattermost.atlassian.net/browse/MM-41036

#### Checklist
- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [x] Unit tests updated

### CODE REVIEW

**Review by cpoile — APPROVED** [review]
Cool!

**Review by calebroseland — APPROVED** [review]

File: tests-e2e/cypress/support/api/user.js [hunk 1]
```diff
@@ -247,6 +247,7 @@ Cypress.Commands.add('apiCreateUser', ({
         if (hideOnboarding) {
             cy.apiSaveOnboardingPreference(createdUser.id, 'hide', 'true');
             cy.apiSaveOnboardingPreference(createdUser.id, 'skip', 'true');
+            cy.apiSaveUserPreference([{user_id: createdUser.id, category: 'crt_tutorial_triggered', name: createdUser.id, value: '999'}], createdUser.id);
```
Comment (Phrynobatrachus) [inline tests-e2e/cypress/support/api/user.js:250]: Can you put this in `apiDisableTutorials` instead please? Or if there's enough overlap between `hideOnboarding` and `bypassTutorial`, maybe all of that could be combined?

**Review by Phrynobatrachus — CHANGES_REQUESTED** [review]

File: tests-e2e/cypress/support/api/user.js (hunk 1 above)
Comment (lieut-data) [reply tests-e2e/cypress/support/api/user.js]: Ooh, good catch -- much cleaner! I opted to inline into `apiDisableTutorials` for now.

**Review by Phrynobatrachus — APPROVED** [review]

**Pull request merged**